ros/api/v1/hosts.py

```diff
@@ -412,9 +412,18 @@ class ExecutiveReportAPI(Resource):
     def get(self):
         account_number = identity(request)['identity']['account_number']
         system_queryset = system_ids_by_account(account_number, fetch_records=True)
+        systems_with_performance_record_query = (
+            db.session.query(PerformanceProfile, System, RhAccount)
+            .join(System, System.id == PerformanceProfile.system_id)
+            .join(RhAccount, RhAccount.id == System.account_id)
+            .filter(
+                PerformanceProfile.system_id.in_(
+                    system_ids_by_account(account_number).subquery())
+            )
+        )
```

Comment:
Can we query something like this?

```suggestion
        systems_with_performance_record_query = (
            (db.session.query(PerformanceProfile.system_id)
            .filter(PerformanceProfile.system_id.in_(system_ids_by_account(account_number).subquery())))
            )
        )
```

Reply:
Good catch! :+1: 
Yes, it will be better than loading all columns from these three tables as don't require any additional info.